plugins/aea-test-autonomy/aea_test_autonomy/base_test_classes/agents.py

```diff
@@ -96,6 +96,9 @@ class BaseTestEnd2End(AEATestCaseMany, UseFlaskTendermintNode):
     ledger_id: str = "ethereum"
     key_file_name: str = "ethereum_private_key.txt"
     USE_GRPC = False
+    # usually test envs provide the full set of dependencies; only select
+    # when you want to run the installation of the packages as part of the test case.
+    RUN_AEA_INSTALL = False
```

Comment:
Without this tests fail. We should not be installing in tests anyway by default imo.